core/platform/taskqueue/cloud_taskqueue_services.py

```diff
@@ -19,6 +19,7 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
+import datetime # pylint: disable=unused-import
```

Comment:
Should we move this import to the `if MYPY` block to clearly state that this import is needed just for mypy.

Reply:
I did a mistake here. This pylint disable can be removed.
We have 2 options here:
1. Import datetime globally.(Like I did now)
2. Import datetime in if MYPY and enclose the type of datetime in quotes. Like this `'datetime.datetime'`

Reply:
Hmm, again I think that `from __future__ import annotations` might be helpful and then the 2 would be fine to do. But yeah 1 seems fine too if the pylint disable is not needed.

Reply:
Thanks! Done using 1!